modules/transactions.js

```diff
@@ -547,6 +548,10 @@ shared.addTransactions = function (req, cb) {
 			}
 		}
 
+		if (keypair.publicKey.toString('hex') === exceptions.mainnetGenesisPublicKey && modules.system.getHeight() > 1) {
+			return setImmediate(cb, 'Attempt to send the money from genesis address');
+		}
+
```

Comment:
This is the right logic, but imo wrong location. It should be handled within `Transaction.prototype.process` or `Transaction.prototype.verify`.